src/CallHandler.tsx

```diff
@@ -150,14 +150,10 @@ export default class CallHandler extends EventEmitter {
     // call with a different party to this one.
     private transferees = new Map<string, MatrixCall>(); // callId (target) -> call (transferee)
     private audioPromises = new Map<AudioID, Promise<void>>();
-    private dispatcherRef: string = null;
-    private supportsPstnProtocol = null;
-    private pstnSupportPrefixed = null; // True if the server only support the prefixed pstn protocol
-    private supportsSipNativeVirtual = null; // im.vector.protocol.sip_virtual and im.vector.protocol.sip_native
-    private pstnSupportCheckTimer: number;
-    // For rooms we've been invited to, true if they're from virtual user, false if we've checked and they aren't.
-    private invitedRoomsAreVirtual = new Map<string, boolean>();
-    private invitedRoomCheckInProgress = false;
+    private dispatcherRef: string | null = null;
+    private supportsPstnProtocol = false;
+    private pstnSupportPrefixed = false; // True if the server only support the prefixed pstn protocol
+    private supportsSipNativeVirtual = false; // im.vector.protocol.sip_virtual and im.vector.protocol.sip_native
```

Comment:
These started off as null because we only have a definitive yes/no answer after we've checked with the server, so we're losing this info.

Reply:
The question is: do we need this info, or just falling back by default to it being turned off isn't enough?